tests/test_things_example.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import Mock
+
+from examples.things import app, things, ThingsResource
+import falcon
+
+
+def test_app():
```

Comment:
If testing like I describe below, I don't think this adds substantial value. We  will test the routing is connected properly by `assert`ing on the result of the simulated request.